external/storm-kafka/src/jvm/storm/kafka/DynamicBrokersReader.java

```diff
@@ -41,6 +41,14 @@
     private String _topic;
 
     public DynamicBrokersReader(Map conf, String zkStr, String zkPath, String topic) {
+        // Check required parameters
+        if(conf == null) {LOG.error("conf cannot be null");}
```

Comment:
@harshach Agreed. I didn't take that approach because it would require significantly more changes to the spout since `DynamicBrokerReader` doesn't reference `KafkaConfig` or `SpoutConfig` currently. Also it would require a modification to `DynamicBrokerReader`'s constructor to accept a `KafkaConfig` instance breaking legacy code. I thought that would be a big change for a minor storm release.